src/core/tsi/alts/zero_copy_frame_protector/alts_zero_copy_grpc_protector.h

```diff
@@ -23,33 +23,37 @@
 
 #include <stdbool.h>
 
-#include "src/core/tsi/transport_security_grpc.h"
+#include "src/core/tsi/transport_security_interface.h"
 
-///
-/// This method creates an ALTS zero-copy grpc protector.
-///
-///- key: a symmetric key used to seal/unseal frames.
-///- key_size: the size of symmetric key.
-///- is_rekey: use rekeying AEAD crypter.
-///- is_client: a flag indicating if the protector will be used at client or
-///  server side.
-///- is_integrity_only: a flag indicating if the protector instance will be
-///  used for integrity-only or privacy-integrity mode.
-///- enable_extra_copy: a flag indicating if the protector instance does one
-///  extra memory copy during the protect operation for integrity_only mode.
-///  For the unprotect operation, it is still zero-copy. If application intends
-///  to modify the data buffer after the protect operation, we can turn on this
-///  mode to avoid integrity check failure.
-///- max_protected_frame_size: an in/out parameter indicating max frame size
-///  to be used by the protector. If it is nullptr, the default frame size will
-///  be used. Otherwise, the provided frame size will be adjusted (if not
-///  falling into a valid frame range) and used.
-///- protector: a pointer to the zero-copy protector returned from the method.
-///
-/// This method returns TSI_OK on success or a specific error code otherwise.
-///
+//
+// This method creates an ALTS zero-copy grpc protector.
+//
+//- key: a symmetric key used to seal/unseal frames. The given buffer may be
+//  mutated if copy_key is set to false.
+//- key_size: the size of symmetric key.
+//- copy_key: when true, the given key will not be mutated and a copy of it
+//  will be made for the underlying encrypter. If false, the caller must ensure
```

Comment:
micro-nit: s/encrypter/crypter (for consistency with the rest of the language)

Reply:
Done.